pkg/config/containers.conf

```diff
@@ -157,7 +157,7 @@ default_sysctls = [
 
 # Logging driver for the container. Available options: k8s-file and journald.
 #
-# log_driver = "journald"
+log_driver = "journald"
```

Comment:
I thought we were not going to do this?  Since a distribution could accidently pick this up.  I think we should stick to this being disabled and just turn it on in Rawhide and RHEL9 Beta, once we have a release in F35 we can look at making it the default here.

Reply:
Roger that. I'll create a follow-up PR.